EIPS/eip-7621.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: 7621
```

Comment:
```suggestion
eip: 7623
```
Assigning next sequential EIP/ERC/RIP number.  EIP/ERC/RIP numbers are issued from a single set of numbers.  7621 was already issued to an ERC yesterday.
Numbering changed to sequential from 7500 and is no longer the PR number.  

Please also update the filename.

Reply:
Updated